ibridges/meta.py

```diff
@@ -35,21 +35,33 @@ class MetaData:
     """
 
     def __init__(
-        self, item: Union[irods.data_object.iRODSDataObject, irods.collection.iRODSCollection]
+        self,
+        item: Union[irods.data_object.iRODSDataObject, irods.collection.iRODSCollection],
+        blacklist: str = r"^org_*",
```

Comment:
That might be a bit too general. It is necessary for Yoda, but other irods instances might use org_* for perfectly harmless and evennecessary items, e.g. org_number for a project number .... For an experimental feature it is ok, but we need a different approach when we release the software as a package.

Reply:
I'm open to suggestions, but with the current state, I don't think we'll have a lot of options.